ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -0,0 +1,188 @@
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+import {withRouteData} from 'app/components/app-router';
+import {Button} from 'app/components/buttons';
+import {FadeBox} from 'app/components/containers';
+import {FlexColumn, } from 'app/components/flex';
+import {Arrow, ClrIcon, withCircleBackground} from 'app/components/icons';
+import {RadioButton} from 'app/components/inputs';
+import {AoU} from 'app/components/text-wrappers';
+import {withProfileErrorModal} from 'app/components/with-error-modal';
+import {styles} from 'app/pages/profile/profile-styles';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {
+  daysFromNow,
+  displayDateWithoutHours,
+  useId,
+  withStyle
+} from 'app/utils';
+import {profileStore, useStore} from 'app/utils/stores';
+
+const {useState} = React;
+const LOOKBACK_PERIOD = 330;
+
+const renewalStyle = {
+  h1: {
+    fontSize: '0.83rem',
+    fontWeight: 600,
+    color: colors.primary
+  },
+  h2: {
+    fontSize: '0.75rem',
+    fontWeight: 600
+  },
+  h3: {
+    fontSize: '0.675rem',
+    fontWeight: 600
+  },
+  card: {
+    backgroundColor: colors.white,
+    border: `1px solid ${colorWithWhiteness(colors.dark, 0.8)}`,
+    borderRadius: '0.2rem',
+    display: 'flex',
+    fontSize: '0.58rem',
+    fontWeight: 500,
+    height: 345,
+    lineHeight: '22px',
+    margin: 0,
+    padding: '0.5rem',
+    width: 560
+  }
+};
+
+const BackArrow = withCircleBackground(() => <Arrow style={{height: 21, width: 18}}/>);
+
+const RenewalCard = withStyle(renewalStyle.card)(
+  ({step, title, TitleComponent, lastCompletion, nextReview, children, style}) => {
+    const daysRemaining = daysFromNow(nextReview);
+    return <FlexColumn style={style}>
+      <div style={renewalStyle.h3}>STEP {step}</div>
+      <div style={renewalStyle.h3}><TitleComponent/></div>
+      <div style={{ color: colors.primary, margin: '0.5rem 0', display: 'grid', gridTemplateColumns: '6rem 1fr'}}>
+        <div>Last Updated On:</div>
+        <div>Next Review:</div>
+        <div>{`${displayDateWithoutHours(lastCompletion)}`}</div>
+        <div>{`${displayDateWithoutHours(nextReview)} (${daysRemaining > 0 ? daysRemaining + ' days' : 'expired'})`}</div>
```

Comment:
Doesn't this suffer from the same bug you resolved in the other file? Should this be `>= 0`? Showing `0 days remaining` on the day of (but before) expiration seems correct to me.